packages/gestalt/src/Table.css

```diff
@@ -29,3 +29,11 @@
 .tbody tr:not(:last-child) td {
   border-bottom: 1px solid var(--g-colorGray150);
 }
+
+.hoverShadeGray {
+  background-color: var(--g-colorGray150);
```

Comment:
@faisalgedi this should be `#efefef` or `--g-colorGray100` instead of `--g-colorGray150`